### DISCUSSION TITLE
Homematic

### DISCUSSION THREAD

**Discussion by mt6557** [rdmtc/RedMatic#547] [General]
Hallo, ist der Homematic Kapazitiver Füllstandsmesser HM-Sen-Wa-Od als Geräte in RedMatic vorhanden? Ich bekommen ihn nicht über RedMatic in Homekit angezeigt. Er wird auch nicht als Geräte im Node dargestellt.

Vielen Dank für eine Antwort.

**Comment by hobbyquaker** [comment]
Geräteunterstützung in HomeKit ist Thema von RedMatic-HomeKit; Version 4.0 (in Entwicklung, für RedMatic 9) erkennt Geräte generisch über die Kanalbeschreibung. Bitte dort prüfen bzw. melden: https://github.com/rdmtc/RedMatic-HomeKit. Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
